### DISCUSSION TITLE
Instruction does not match screen and I don't know how to continue: Edit index.md

### DISCUSSION THREAD

**Discussion by DDMAIN** [skills/.github#322] [Communicate using Markdown] [answered]
Hi, i am following this tutorial and when I get to instruction 6 the **Edit file** option appears disabled. 

### :keyboard: Activity: Edit your file with headers

1. Open a new browser tab, and work on the steps in your second tab while you read the instructions in this tab.
1. Open the **pull requests** tab.
1. Click **New pull request**, for the branches to compare, select `base: main` and `compare: start-markdown`.
1. Click **Create pull request**.
1. In this pull request, go to the **Files changed** tab. We made an empty file `index.md` for you.
1. Select **Edit file** from the three dotted **...** menu in the upper right corner of the file view on `index.md`.

![imagen](https://github.com/skills/.github/assets/69592236/834903dd-4ca9-4a9b-9b71-d67537338108)

How can I continue?

**Comment by sinsukehlab** [comment, marked as answer]
You should first create the pull request (task 4) before editing a file in the pull request.
You can't edit a file while comparing branches.

**Reply by DDMAIN** [reply]
Oh with "**Create pull request**" I thought he meant just hitting the button the first time instead of doing the whole process.

Ok, thank you so much.

**Reply by sinsukehlab** [reply]
I'm glad that I could be of help.
